### ISSUE TITLE
Disparity in accuracy

### ISSUE THREAD

**Issue by denabazazian** [MingyeXu/GS-Net#4]
Thanks for sharing your code. 
I have tested the model by the provided pretrained model in the repository and I got the following accuracy: 

> test acc: 0.928687, test avg acc: 0.900413

But, when I tested the model after training 500 epochs, the accuracy was lower than the one from the pretrained model: 

>  test acc: 0.926661, test avg acc: 0.889669

This is what I got from the pretrained model: 
```
$ python main.py --eval True --model_path 'pretrained/model_1024_92.9.t7'
Namespace(batch_size=64, dataset='modelnet40', dropout=0.5, emb_dims=1024, epochs=500, eval=True, exp_name='exp', k=20, lr=0.001, model='GSNET', model_path='pretrained/model_1024_92.9.t7', momentum=0.9, no_cuda=False, num_points=1024, seed=1, test_batch_size=32, use_sgd=False)
Using GPU : 0 from 1 devices
Test :: test acc: 0.928687, test **avg acc: 0.900413**
```

This is what I got from the model that I have trained by your code: 
```
$ python main.py --eval True --model_path 'checkpoints/exp/models/model.t7'
Namespace(batch_size=64, dataset='modelnet40', dropout=0.5, emb_dims=1024, epochs=500, eval=True, exp_name='exp', k=20, lr=0.001, model='GSNET', model_path='checkpoints/exp/models/model.t7', momentum=0.9, no_cuda=False, num_points=1024, seed=1, test_batch_size=32, use_sgd=False)
Using GPU : 0 from 1 devices
Test :: test acc: 0.926661, test **avg acc: 0.889669**
```
It looks like that everything in the namespace is the same such as emb_dims =1024. Do you know why this disparity happens particularly in avg_acc?

**Comment by sheshap** [comment]
I trained from scratch and got Test :: test acc: 0.930713, test avg acc: 0.895570

**Comment by sheshap** [comment]
Weird

no matter how many times I run I get below results,

$ python main.py --eval True --model_path 'checkpoints/exp/models/model.t7'
Namespace(batch_size=64, dataset='modelnet40', dropout=0.5, emb_dims=1024, epochs=500, eval=True, exp_name='exp', k=20, lr=0.001, model='GSNET', model_path='checkpoints/exp/models/model.t7', momentum=0.9, no_cuda=False, num_points=1024, seed=1, test_batch_size=32, use_sgd=False)
Using GPU : 0 from 2 devices
Test :: test acc: 0.930713, test avg acc: 0.895570

**Comment by denabazazian** [comment]
It's weird, because I always get: `Test :: test acc: 0.926661, test **avg acc: 0.889669**`
Since I am using exactly the same hyper-parameters and the same model, I am just wondering to know what has made this disparity, although the difference is not that much.
